### DISCUSSION TITLE
[需求] vscode本地小说文件点击章节后在右边打开新窗口

### DISCUSSION THREAD

**Discussion by TAnsz** [aooiuu/any-reader#50] [Q&A]
### 功能描述

现在是在左边的边栏直接打开了，不方便切换章节，显示内容也过少。
希望能在右边打开新的标签页用于查看。
![image](https://github.com/user-attachments/assets/95ab7deb-31f7-4df4-a865-211000a0a638)

**Comment by aooiuu** [comment]
可以先点击**侧边栏**底部的**大搜索**图标，这会在右侧**编辑器**区域打开一个**Tab**，在新打开的页面里面操作就可以了
